thirdparty/CMakeLists.txt

```diff
@@ -172,19 +168,19 @@ endif()
 #
 
 if(ENABLE_SAMPLES OR ENABLE_TESTS)
-    find_package(ZLIB QUIET NO_MODULE)
+    find_package(ZLIB QUIET)
     if(ZLIB_FOUND)
-        # need to make it global to use outside of the current sub-directory
-        set_target_properties(ZLIB::ZLIB PROPERTIES IMPORTED_GLOBAL ON)
+        # FindZLIB module defines ZLIB::ZLIB, no extra steps are required
     endif()
 
     # cmake has failed to find zlib, let's try pkg-config
     if(NOT ZLIB_FOUND AND PkgConfig_FOUND)
         pkg_search_module(zlib QUIET
-                          IMPORTED_TARGET GLOBAL
```

Comment:
those changes you are reffering to, also renames CMakeLists.txt into dependencies.cmake and use include() instead of add_sudirectory. 
So, porting the changes as is may not work as expected. 

With current changes ZLIB::ZLIB is bot visible outside of thrdpiarty folder.

Reply:
Thank you for your comment! You are right, this cmake change might not work as expected. I will close this PR as we found a workaround for the Ubuntu20 static build. We will use this workaround until future update to OV2023.1 for vpux-plugin.